### PR TITLE
little modifications

### PR DESCRIPTION
**Pull request by almereyda**
- reverse entitled naming order
- DRY
- correctly guess .mp4 extensions

### CODE REVIEW

File: bin/backup-all-my-flickr-photos [hunk 1]
```diff
@@ -252,7 +252,7 @@ def guess_extension(url, headers):
             if extension in _extensions:
                 return extension
     if '/play/' in url.lower():
-        return 'mp4'
+        return '.mp4'
```
Comment (Flimm) [inline bin/backup-all-my-flickr-photos:255]: Nice catch!

Comment (Flimm) [reply bin/backup-all-my-flickr-photos:255]: I just realised this is the same fix as #19

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 /backup_all_my_flickr_photos.egg-info
 /dist
 /build
+/Download*
```
Comment (Flimm) [inline .gitignore:4]: What is this folder and why does it need to be added to `.gitignore`?

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+.python-version
```
Comment (Flimm) [inline .gitignore:5]: If the user is using `pyenv`, they can ignore `.python-version` by listing it in `.git/info/exclude`, which isn't committed to the repo. There is no need to add it to the repo that is shared by everyone.

**Comment by Flimm** [comment]
Thanks for sharing your changes! Your bug fix regarding `.mp4` is valuable. (It turns out that this bug fix was already submitted in another pull request). The other changes don't look significant to me, and I prefer to keep changes minimal since I don't have much time to maintain this package nowadays.

**Comment by almereyda** [comment]
I agree! I think the `.mp4` one was the only real error in this proposal, and the other changes merely cosmetic.

**Pull request closed without merging**